src/state/index.ts

```diff
@@ -43,10 +43,10 @@ const store = configureStore({
   preloadedState: load({ states: PERSISTED_KEYS, disableWarnings: process.env.NODE_ENV === 'test' }),
 })
 
-store.dispatch(updateVersion())
-
 setupListeners(store.dispatch)
 
+store.dispatch(updateVersion())
```

Comment:
What is the goal here?

`setupListeners(store.dispatch)` sets up the RTK-Query listeners for on window focus/blur and such.